rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/BakeRequest.groovy

```diff
@@ -56,14 +56,15 @@ class BakeRequest {
   String ami_name
   String ami_suffix
   Boolean upgrade
+  String flavor //openstack only
```

Comment:
Would like to try to avoid introducing (more) provider-specific attributes in common areas. Please change this to something generic like `machine_type` or 'instance_type`, and drop the comment. This seems to be the intent of this attribute based on the description here: http://docs.openstack.org/openstack-ops/content/flavors.html